test/e2e/config/azure-dev.yaml

```diff
@@ -86,9 +86,15 @@ providers:
       targetName: "cluster-template-machine-pool-windows.yaml"
     - sourcePath: "${PWD}/templates/test/ci/cluster-template-prow-external-cloud-provider.yaml"
       targetName: "cluster-template-external-cloud-provider.yaml"
+    - sourcePath: "${PWD}/templates/test/ci/cluster-template-prow-aks-multi-tenancy.yaml"
+      targetName: "cluster-template-aks-multi-tenancy.yaml"
+    replacements:
```

Comment:
what's this? rebase errata from #1332 ?

Reply:
I noticed it in my PR (cluster identity) as well after I rebased, didn't realize it should be removed

Reply:
I actually thought this was needed by the e2e framework. I'll remove it if not needed.

Reply:
I think it came from the rebase, I had the same happen, its not in the file in the `main` branch

Reply:
yeah the replacements were moved above inside the manager in another PR (which is where they should have been since the beginning)

Reply:
Oh my bad, I was thinking you were talking about 
```
    - sourcePath: "${PWD}/templates/test/ci/cluster-template-prow-aks-multi-tenancy.yaml"      
        targetName: "cluster-template-aks-multi-tenancy.yaml"
```

Let me remove the replacements block.